build.gradle.kts

```diff
@@ -27,14 +27,7 @@ buildscript {
     }
 
     dependencies {
-        // R8 below 3.1.79, 3.2.70, and 3.3.62, has an issue;
-        // it tries to convert some Enums to ints but kind of stops halfway.
-        // This was previously solved by pinning R8 to 3.3.62,
-        // but that requires Google's own R8 repo as this version wasn't on Maven,
-        // which is an issue since F-Droid decided to not allowlist that repo.
-        // Instead, we now use Android Gradle Plugin 7.3.0-beta05, which has a fix.
-        // https://issuetracker.google.com/issues/236618700
-        classpath("com.android.tools.build:gradle:7.3.0-beta05")
+        classpath("com.android.tools.build:gradle:7.3.1")
```

Comment:
Should this be committed separately maybe?

Reply:
But it is though

Reply:
I meant outside the PR, but whatever.

Reply:
Oh I now notice your other PR, sure